CHANGELOG.md

```diff
@@ -29,6 +29,11 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### Linter
 
+#### New features
```

Comment:
i add some changelogs here,and also add the changelogs for my previous pr which forget.